src/components/modal/modal.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Modal as DefaultModal } from 'antd';
+
+import './styles.scss';
+
+export const Modal = ({
+  isOpen,
+  onCancel,
+  onOk,
+  header,
+  warning,
+  children,
+}) => {
+  return (
+    <DefaultModal visible={isOpen} onCancel={onCancel} onOk={onOk}>
+      <div>{header}</div>
+      <div className='modal-header'></div>
+      {children}
+      {warning && (
```

Comment:
Explain why warning says `You must fill in all the fields`

Reply:
To fill in each field.